public/assets/js/project.js

```diff
@@ -1,14 +1,60 @@
-function passwordFunction() {
-  document.getElementById('forgotPassLink').innerHTML =
-    'Thank you. A link has been sent to your email to reset your username.';
-}
-
-function userFunction() {
-  document.getElementById('forgotUserLink').innerHTML =
-    'Thank you. A link has been sent to your email to reset your password.';
-}
-
-function regFunction() {
-  document.getElementById('regisConfirmation').innerHTML =
-    'Thank You. Registration is complete and added to database.';
-}
+$(document).ready(function() {
+  console.log('file loaded');
+
+  function passwordFunction() {
+    document.getElementById('forgotPassLink').innerHTML =
+      'Thank you. A link has been sent to your email to reset your username.';
+  }
+
+  function userFunction() {
+    document.getElementById('forgotUserLink').innerHTML =
+      'Thank you. A link has been sent to your email to reset your password.';
+  }
+
+  function regFunction() {
+    document.getElementById('regisConfirmation').innerHTML =
+      'Thank You. Registration is complete and added to database.';
+  }
+
+  $('#hrEmployeeDashboard').on('click', function() {
+    const access = $('#whoami').data("access");
+    $.ajax({
+      method: 'GET',
+      url: '/api/employees?access=' + access
+    }).then(function(data) {
+      for (let i = 0; i < data.length; i++) {
+        const row = $(`<tr class="employeeData" data-emp="${data[i].id}">`);
+        const firstName = $(`<td>${data[i].first_name}</td>`);
+        const lastName = $(`<td>${data[i].last_name}</td>`);
+        const address = $(`<td>${data[i].address}</td>`);
+        const startDate = $(`<td>${data[i].start_date}</td>`);
+        const salary = $(`<td>${data[i].annual_salary}</td>`);
+        const title = $(`<td>${data[i].role}</td>`);
+        const schedule = $(`<td>${data[i].weekly_schedule}</td>`);
+        const shift = $(`<td>${data[i].scheduled_shift}</td>`);
+
+        row.append(firstName, lastName, address, startDate, salary, title, schedule, shift);
+
+        $('#employeeList').append(row);
+      }
+
+      refreshEmployeeClickListener();
+    });
+  });
+
+  function refreshEmployeeClickListener() {
+    console.log('ran this func');
+    $('.employeeData').on('click', function() {
```

Comment:
This is entirely optional, and I wouldn't recommend spending a lot of time on it unless y'all have a lot of work somewhere else that I didn't see.

But what you could do with it is use `window.location.href = '/some/other/page';` to redirect you to a page with a form, for example, that has the clicked employee's data pre-filled, and allows the HR rep to change things (like promotions, salary, schedule, etc), or delete the employee if they were fired.  The submit button would send the form as a PUT to the API, which would update the database, and when the table is pulled up again in the modal, that employee's data would be updated.  For delete, you would want a separate button that sends a DELETE request to the API with that employee's ID and the API will remove the user from the database, and you won't see that user in the modal the next time you click "Employee Dashboard".

This would require a couple new API routes in the employee router, as well as another route (probably in index router) that renders another handlebars template with the form.  That second route would need to make a sequelize call to the employees table to get that employee's data and pass it to `res.render()`.  Then you'd remove the ajax call inside this on click function, because all of that would be happening on the server.